AppCenterPush/AppCenterPush/MSPush.m

```diff
@@ -118,6 +118,14 @@ + (void *)userNotificationCenterDelegateContext {
 
 #endif
 
+#pragma mark - MSUserIdContextDelegate
+- (void) userIdContext:(MSUserIdContext *)__unused userIdContext didUpdateUserId:(NSString *)__unused userId {
```

Comment:
use this `userId` instead of getting from singleton

Reply:
Empty line before

Reply:
@MatkovIvan I was actually going to remove parameters since they are not used at all

Reply:
Please do not remove it, and use it instead of getting from singleton